Plugins/Rename/Rename.cpp

```diff
@@ -532,6 +540,26 @@ void Rename::SendNameUpdate(CNWSCreature *targetCreature, PlayerID observerPlaye
     }
 }
 
+bool Rename::IsCreatureInLastUpdateObjectList(CNWSPlayer *player, ObjectID creatureId)
+{
+    auto *lastUpdateObj = player->GetLastUpdateObject(creatureId);
+    if (lastUpdateObj)
+        return true;
+
+    auto *partyObjectsList = player->m_pActivePartyObjectsLastUpdate->m_pcExoLinkedListInternal;
+    if (!partyObjectsList)
+        return false;
+
+    for (auto *head = partyObjectsList->pHead; head; head = head->pNext)
+    {
+        auto *partyMember = static_cast<CLastUpdatePartyObject *>(head->pObject);
+        if (partyMember && partyMember->m_nPlayerId == creatureId)
```

Comment:
You're comparing creatureId (oid) with a player ID here.

Reply:
`m_nPlayerId` is actually an object id. Looks like the field name is confusing.

Reply:
Are you sure? I think `m_oidNWSObject` (currently controlled object) and `m_oidPCObject` (primary PC object) are the OIDs, and `m_nPlayerId` is the `CNetLayer` ID

Reply:
Waaait, idiot. This is a LUPO field, I was thinking of CNWSPlayer/CNWSClient. You're right, sorry!

Reply:
Yeah I was looking for an object id field and was surprised that it was called this way. :) It’d be nice to rename it in the game, but not sure how risky that’d be.